client/src/components/Dialogs/CartModal.js

```diff
@@ -0,0 +1,264 @@
+                      <TableCell align="justify">{recipe.name}</TableCell>
+                      <Hidden smDown>
+                        <TableCell align="justify" nowrap="nowrap">
+                          <span>{qty}</span>{" "}
+                          <ButtonGroup
+                            disableElevation
+                            variant="contained"
+                            className={classes.btnGroup}
+                          >
+                            <Button
+                              onClick={() =>
+                                qty > 1
+                                  ? decreaseCount(dispatch, {
+                                      id: recipe._id,
+                                      qty: qty - 1,
+                                      chef: recipe.user,
+                                    })
+                                  : removeFromCart(dispatch, recipe._id)
+                              }
+                            >
+                              -
+                            </Button>
+                            <Button
+                              onClick={() =>
+                                increaseCount(dispatch, {
+                                  id: recipe._id,
+                                  qty: qty + 1,
+                                  chef: recipe.user,
+                                })
+                              }
+                            >
+                              +
+                            </Button>
+                          </ButtonGroup>
+                        </TableCell>
+                      </Hidden>
+                      <Hidden smDown>
+                        <TableCell align="right">{`${
+                          recipe.price
+                        }x${qty}=${addDecimals(
+                          qty * recipe.price
+                        )}`}</TableCell>
+                      </Hidden>
+                      <TableCell
+                        align="right"
+                        onClick={(e) => handleDelete(recipe._id)}
+                      >
+                        <IconButton edge="end" aria-label="delete">
+                          <DeleteForeverIcon />
+                        </IconButton>
+                      </TableCell>
+                    </TableRow>
+                  ))}
+                </TableBody>
+              </Table>
+            </TableContainer>
+          </ListItem>
+          <ListItem>
+            <ListItemText>
+              <Typography gutterBottom variant="h6">
+                Total Price
+              </Typography>
+            </ListItemText>
+            <ListItemText>
+              <Typography gutterBottom variant="h6">
+                {" "}
+                {addDecimals(
+                  cart.reduce(
+                    (acc, cur) => acc + cur.recipe.price * cur.qty,
+                    0
+                  ),
+                  2
+                )}
```

Comment:
This is a nitpick, but it would be good to add a $ here

Reply:
Hey @dproc96 , I have added a $ in one line before this, that's why this comment is not outdated I guess.

Also, I wanted to say that the Test components were not for testing actually, the thing is I was working on recipe and chef profile for this ticket and also for my previous ticket but the actual components were  not written yet, so I had to write a simple Temporary component to do my tickets.  Last night, I finally got the actual components and integrated my code to them. We will have another refactoring in recent future, I thought since I was working with the Test components for the most part, I keep them for now, in case I forgot some edge cases that I took care of before. But after that refactoring, I will get rid of the TestComponents or rather TemporaryComponents completely.